README.md

```diff
@@ -1,24 +1,16 @@
 <!-- default badges list -->
-![](https://img.shields.io/endpoint?url=https://codecentral.devexpress.com/api/v1/VersionRange/260013606/22.1.3%2B)
 [![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/T907025)
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
-# Blazor - How to reuse a component and configure its state via a dictionary
+# Grid for Blazor - How to use a dictionary to configure the component state
 
-In your application you have multiple components. For instance, let it be Grid. Some of the grids have one set of settings-other grids - other settings:
+This example demonstrates how you can use a dictionary to configure the [Blazor Grid](https://docs.devexpress.com/Blazor/403143/components/grid) component. This approach allows you to configure multiple components based on one set of settings.
```

Comment:
1. Approach.
2. Set of settings | or settings set, for that matter. It does not sound very well.

```suggestion
This example demonstrates how you can use a dictionary to configure the [Blazor Grid](https://docs.devexpress.com/Blazor/403143/components/grid) component. This technique allows you to configure multiple components based on one options set.
```